configure

```diff
@@ -486,13 +498,23 @@ if [ -z "$FLINT_LIB" ]; then
          EXTRA_SHARED_FLAGS="$EXTRA_SHARED_FLAGS -compatibility_version $FLINT_MAJOR.$FLINT_MINOR"
          EXTRA_SHARED_FLAGS="$EXTRA_SHARED_FLAGS -current_version $FLINT_MAJOR.$FLINT_MINOR.$FLINT_PATCH"
          ;;
-      CYGWIN* | MINGW*)
+      CYGWIN*)
+         FLINT_LIBNAME="libflint.dll"
+         FLINT_LIB="libflint-$FLINT_MAJOR.dll"
+         EXTRA_SHARED_FLAGS="-static-libgcc"
+         EXTRA_SHARED_FLAGS="$EXTRA_SHARED_FLAGS -shared"
+         EXTRA_SHARED_FLAGS="$EXTRA_SHARED_FLAGS -Wl,--export-all-symbols"
+         EXTRA_SHARED_FLAGS="$EXTRA_SHARED_FLAGS -Wl,-soname,libflint-$FLINT_MAJOR.dll.$FLINT_MINOR.$FLINT_PATCH"
```

Comment:
```suggestion
         EXTRA_SHARED_FLAGS="$EXTRA_SHARED_FLAGS -Wl,-soname,libflint-$FLINT_MAJOR.dll"
```

Reply:
If the line numbers here are correct, this changes only Cygwin. Should I make the same change for MinGW as well?

Reply:
On second thought, this whole line is unnecessary.